AWSCore/Bolts/AWSTask.m

```diff
@@ -10,13 +10,13 @@
 
 #import "AWSTask.h"
 
-#import <libkern/OSAtomic.h>
+#import <stdatomic.h>
```

Comment:
what is this change about?

Reply:
`OSAtomic` is now deprecated, and we should use `stdatomic` instead